src/vdaf.rs

```diff
@@ -105,6 +147,9 @@ pub trait Vdaf: Clone + Debug {
     /// The number of Aggregators. The Client generates as many input shares as there are
     /// Aggregators.
     fn num_aggregators(&self) -> usize;
+
+    /// Construct contextual parameters for decoding this VDAF's input share messages.
+    fn input_share_decoding_parameter(&self, is_leader: bool) -> Self::InputShareDecodingParameter;
```

Comment:
Not every VDAF necessarily has a notion of "leader", so it would be better to not bleed this through the API. However this method won't be needed if you take my suggestion above.